### PR TITLE
Pathology feedback updates

### LINKED ISSUE #95: Pathology data: reorder plots
Request that the CONTROL be shown on the LEFT and the MODEL be shown on the RIGHT

### LINKED ISSUE #96: Pathology data: fields to exclude
These fields found in the 5x data should be excluded:
Astrocyte Cell Size (GFAP)
Microglia size (IBA1)
NeuN Cell Density

### LINKED ISSUE #97: Pathology data: rename one field for 3xTg-AD
The 3xTg-AD strain has a field labeled "Microglia Cell Density". 
This should be updated to "Microglia Cell Density (IBA1)" so it's consistent with 5x.

### LINKED ISSUE #98: Pathology data: scale plots by phenotype AND model
Is it possible to update the logic in how axes are scaled, such that each model-CTRL combo can be scaled separately?
For example, in this screenshot, 5xFAD has much smaller values than 3xTg, and it would be nice to see these expanded. 
Screen Shot 2022-05-11 at 10 08 52 AM

### LINKED ISSUE #99: MODEL-AD vs Model AD
After suggesting a global find-replace, I realized that there are some places we should be using the acronym (MODEL-AD) and some places we should be using the explorer name (Model AD). 

Please see this screenshot for the places that should be reverted to MODEL-AD:

Screen Shot 2022-05-11 at 10 25 26 AM

### LINKED ISSUE #102: Update versions of pathology data files
The following files have been updated (some outliers removed):

syn22101767
syn22049816
syn25913418

### PR DESCRIPTION
**Pull request by sharlagelfand**
Addressing the round of feedback from the pathology team:

Closes #95: Control is now shown on the left and model on the right

Closes #96: Astrocyte cell size, microglia size, and neun cell density removed from 5xFAD data

Closes #97: Names Microglia Cell Density (IBA1) consistently between 5x and 3tg

Closes #99: MODEL-AD used in specific places 

Closes #102: Updated version of data files are used (the team removed some outliers and updated)

Closes #98: The y-axis scale now varies between models/rows (e.g. 5xfad vs 3tg) but is consistent within a model/row (e.g. c57bl/6j and 5xfad have the same y-axis)

### CODE REVIEW

**Comment by j-hendrickson-sage** [comment]
Can we look into the failure of "R-CMD-check / windows-latest (release)"?  OpenSSL seems to be having trouble in the Windows environment.

File: R/magora_boxplot.R [hunk 1]
```diff
@@ -1,17 +1,39 @@
...
 #' @return A ggplot2 object.
 #' @export
-magora_boxplot <- function(data, type = "ggplot2", facet = TRUE, save_name, use_theme_sage = TRUE) {
+magora_boxplot <- function(data, mouse_model_groups, use_theme_sage = TRUE) {
 
-  # Check arguments
-  if (!type %in% c("ggplot2", "plotly")) {
-    stop("`type` should be one of 'ggplot2' or 'plotly'", call. = FALSE)
-  }
+  phenotype_units <- unique(data[["phenotype_units"]])
```
Comment (j-hendrickson-sage) [inline R/magora_boxplot.R:11]: This seems like more pandas style.  dply selects?

**Comment by sharlagelfand** [comment]
> Can we look into the failure of "R-CMD-check / windows-latest (release)"? OpenSSL seems to be having trouble in the Windows environment.

Looks like things are good on this now, I reran the job and it passed. Might just be an intermittent error!

File: R/magora_boxplot.R (hunk 1 above)
Comment (j-hendrickson-sage) [reply R/magora_boxplot.R:11]: Hi Sharla, I'm trying to stay away from opinion, the tidyverse is full of it for better or worse.  I prefer to not mix base R and tidyverse when I can, with preference given to the tidyverse.  You may have a take, too.

**Review by j-hendrickson-sage — APPROVED** [review]
Good!

**Review by j-hendrickson-sage — COMMENTED** [review]
This was most sizable change.  A group meeting may be need to ensure the functionality is correct with Anna Greenwood.

**Pull request merged**